tests/uhttp_ut/uhttp_ut.c

```diff
@@ -1266,6 +1285,32 @@ TEST_FUNCTION(uhttp_client_execute_request_with_content_fails)
     uhttp_client_destroy(clientHandle);
 }
 
+TEST_FUNCTION(uhttp_client_execute_request_multiple_succeed)
```

Comment:
Thanks for adding tests.

Reply:
Opened up a can of worms in the mocked list impl but it was worth it!